msteams-platform/sbs-teams-app-test-tool.yml

```diff
@@ -0,0 +1,351 @@
+
+    1. Select **Default folder**.
+
+        :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/select-default-location.png" alt-text="Screenshot shows the selection of default location.":::
+
+        To change the default location, follow these steps:
+
+        1. Select **Browse**.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/select-browse.png" alt-text="Screenshot shows the selection of browse location option.":::
+
+        1. Select the location for the project workspace.
+        1. Select **Select Folder**.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/select-folder.png" alt-text="Screenshot shows the folder to select.":::
+
+        1. Enter a suitable name for your app and then select the **Enter** key.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/hello-bot.png" alt-text="Screenshot shows where to enter the app name.":::
+
+            A dialog appears, where you need to choose yes or no to trust the authors of the files in this folder.
+
+            :::image type="content" source="~/assets/images/teams-toolkit-v2/first-bot/vsc-trust-authors.png" alt-text="Screenshot shows the dialog to trust or not the authors of the files in this folder.":::
+
+            Now you've successfully created your AI chat bot project workspace.
+
+    ## Take a tour of the bot app source code
+
+    After you finish scaffolding, explore the project directories and files in the Explorer section of the Visual Studio Code.
+    
+    :::image type="content" source="../msteams-platform/assets/images/test-tool/source-code.png" alt-text="Screenshot shows the Teams Toolkit sample bot folder Structure.":::
+    
+    | Folder name | Contents |
+    | --- | --- |
+    | `env/.env.testtool.user` |Configuration file for local environment used by `teamsapp.testtool.yml` to customize the provisioning and deployment rules.  |
+    | `appPackage` | App manifest template files and app icons (color.png and outline.png). |
+    | `appPackage/manifest.json` | App manifest for running the app in local and remote environment.  |
+    |`src/app.js`| Handles business logics for the AI Chat Bot.|
+    |`teamsapp.yml`|This is the main Teams Toolkit project file. The project file defines two primary things:  Properties and configuration Stage definitions. |
+    |`teamsapp.local.yml`|This overrides `teamsapp.yml` with actions that enable local execution and debugging.|
+    |`teamsapp.testtool.yml`|This overrides `teamsapp.yml` with actions that enable local execution and debugging in Test Tool.|
+
+- title: Build and run your AI chat bot app
+  durationInMinutes: 1
+  content: |
+    After you set up your project workspace with Teams Toolkit, build your AI chat bot project. Ensure that you've signed in to your Microsoft 365 account.
+
+    ## Sign in to your Microsoft 365 account
```

Comment:
No need to sigin M365 account for debugging in Test Tool, this section can be removed.

Reply:
Removed this section as suggested.